packages/runed/src/lib/utilities/useMutationObserver/useMutationObserver.svelte.ts

```diff
@@ -0,0 +1,66 @@
+import { IsSupported } from "../IsSupported/IsSupported.svelte.js";
+import { extract } from "../extract/extract.js";
+import { watch } from "../watch/watch.svelte.js";
+import type { MaybeGetter } from "$lib/internal/types.js";
+import { safelyCleanup } from "$lib/internal/utils/effect.svelte.js";
+
+export interface UseMutationObserverOptions extends MutationObserverInit {}
+
+/**
+ * Watch for changes being made to the DOM tree.
+ *
+ * @see https://runed.dev/docs/utilities/useMutationObserver
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver MutationObserver MDN
+ */
+export function useMutationObserver(
+	target: MaybeGetter<HTMLElement | HTMLElement[] | null | undefined>,
+	callback: MutationCallback,
+	options: UseMutationObserverOptions = {}
+) {
+	let observer: MutationObserver | undefined;
+	const isSupported = new IsSupported(() => window && "MutationObserver" in window);
```

Comment:
MutationObserver is very well supported. I think this code is there because, in VueUse, you can pass a custom `window` object. We can drop it.